lib/linalg/scalapack_wrapper.h

```diff
@@ -292,7 +303,9 @@ void qrfactorize(struct QRManager*);
 
 void qaction(struct QRManager*, struct SLPK_Matrix* A, int S, int T);
 
-void qcompute(struct QRManager*);
+void qrcompute(struct QRManager*);
```

Comment:
Several functions in this file need some documentation. It is not obvious what the meanings of `compute` and `factorize` are.